docs/concepts/configuration/assign-pod-node.md

```diff
@@ -190,13 +190,7 @@ flavor and the `preferredDuringSchedulingIgnoredDuringExecution` flavor.
 
 As with node affinity, the legal operators for pod affinity and anti-affinity are `In`, `NotIn`, `Exists`, `DoesNotExist`, `Gt`, `Lt`.
 
-In principle, the `topologyKey` can be any legal label value. However,
-for performance reasons, only a limited set of topology keys are allowed.
-By default the allowed topology keys are:
-
-* `kubernetes.io/hostname`
-* `failure-domain.beta.kubernetes.io/zone`
-* `failure-domain.beta.kubernetes.io/region`
+The `topologyKey` can be any legal label value.
```

Comment:
Well, the available `topologyKey` value set for soft pod AntiAffinity is actually indicated with the scheduler flag option `--failure-domains`.
ref: https://github.com/kubernetes/kubernetes/blob/master/plugin/cmd/kube-scheduler/app/options/options.go#L92

 So I'd suggest not to delete the original description #L193-#L199, but add a few lines to tell how to setup for custom topologyKeys.